sixpack.js

```diff
@@ -199,7 +199,16 @@
         } else {
             var httpModule = url.startsWith('https') ? 'https' : 'http';
             var http = eval('require')(httpModule); // using eval to skip webpack bundling and warnings
-            var req = http.get(url, { headers: { 'Cookie': cookie } }, function(res) {
+
+            const parsedUrl = new URL(url);
+            const options = {
```

Comment:
To avoid syntax error in older browsers
```suggestion
            var parsedUrl = new URL(url);
            var options = {
```